germline_single_sample_workflow.wdl

```diff
@@ -297,3 +321,125 @@ workflow germline_single_sample_workflow {
   }
 }
 
+# Call variants on a single sample with HaplotypeCaller to produce a GVCF
+task HaplotypeCaller {
```

Comment:
So the changes to Haplotypecaller and FilterVCF would go in germline_variant_discovery.wdl